Source/MonoGame.Extended/Maps/Tiled/TiledTileLayer.cs

```diff
@@ -54,22 +54,59 @@ public override void Draw(Camera2D camera)
             {
                 var region = _map.GetTileRegion(tile.Id);
 
-                if (region != null)
-                {
-                    // not exactly sure why we need to compensate 1 pixel here. Could be a bug in MonoGame?
-                    var tx = tile.X * (_map.TileWidth - 1);
-                    var ty = tile.Y * (_map.TileHeight - 1);
-                        
-                    _spriteBatch.Draw(region, new Rectangle(tx, ty, region.Width, region.Height), Color.White);
-                }
+                if(region == null) continue;
```

Comment:
Was just trying to reduce nested code blocks. Try to deal with the negative cases first to avoid nesting. `continue` was what was happening if the `region != null` wasn't true. Not sure of another way to reduce nesting in this case off the top of my head.